controllers/mysqlcluster_controller.go

```diff
@@ -899,6 +899,10 @@ func (r *MySQLClusterReconciler) reconcileV1StatefulSet(ctx context.Context, req
 	podSpec.InitContainers = nil
 	podSpec.WithContainers(containers...)
 	podSpec.WithInitContainers(initContainers...)
+	podSpec.WithSecurityContext(corev1ac.PodSecurityContext().
```

Comment:
This logic has a breaking change.
If a MySQLCluster has a user-defined SecurityContext, moco will not reflect it.
Please merge the settings.

```yaml
apiVersion: moco.cybozu.com/v1beta2
kind: MySQLCluster
metadata:
  name: test
spec:
  podTemplate:
    spec:
      securityContext:         # user-defined securityContext
        seccompProfile:        #
          type: RuntimeDefault #
      containers:
      - name: mysqld
        image: quay.io/cybozu/mysql:8.0.28
    # Specify minimum resources so as not to overwhelm CI resources.
  volumeClaimTemplates:
  - metadata:
      name: mysql-data
    spec:
      accessModes: [ "ReadWriteOnce" ]
      resources:
        requests:
          storage: 1Gi
  logRotationSchedule: "@every 10s"
```

Reply:
Thanks, I fix and add tests.
https://github.com/cybozu-go/moco/pull/443#:~:text=Fix%20pod%20security%20contexts%20not%20being%20merged